compiler/passes/codegen.cpp

```diff
@@ -765,7 +780,7 @@ static void protectNameFromC(Symbol* sym) {
 
 
 // TODO: Split this into a number of smaller routines.<hilde>
-static void codegen_header() {
+static void codegen_header(bool isHeader) {
```

Comment:
Will this function be called multiple times, or could we assume that isHeader will always be true here?